src/components/Dropdown/__tests__/Dropdown.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { render, fireEvent } from "@testing-library/vue";
+import Dropdown from "..";
+
+describe("components/Dropdown", () => {
+  it("should handle a callback on click", async () => {
+    const spy = jest.fn();
+    const { getByTestId } = render(Dropdown, {
+      slots: { default: "Content" },
+      props: {
+        handleChange: spy,
+        value: "opt1",
+        options: [
+          { key: "opt1", label: "Option 1", path: "path1" },
+          { key: "opt2", label: "Option 2", path: "path2" },
+          { key: "opt3", label: "Option 3", path: "path3" },
+        ],
+      },
+    });
+
+    const option1 = getByTestId("option-opt1");
+
+    expect(option1.innerHTML).toEqual("Option 1");
+    await fireEvent(option1!, new Event("click"));
+    expect(spy).toHaveBeenCalled();
```

Comment:
Please check that the correct item is handed to the callback.

Reply:
You can access the parameters of the mocked call through:

`spy.mock.calls[0]`

Reply:
The following line is checking this: